pkg/proto/configuration/global/global.proto

```diff
@@ -294,6 +295,16 @@ message Configuration {
   // Keys of this map correspond to the suffixes of the RLIMIT_*
   // constants (e.g., "NOFILE" to limit the number of file descriptors).
   map<string, SetResourceLimitConfiguration> set_resource_limits = 15;
+
+  // List of compression algorithms supported by the Content Addressable
+  // Storage. When not set, defaults to IDENTITY compression only, which
+  // means that no compression is supported.
+  //
+  // Valid values include:
+  // - IDENTITY: No compression
+  // - ZSTD: Zstandard compression
```

Comment:
REv2 states that IDENTITY is implied. Maybe it's cleaner to stick to that here as well?

> List of compression algorithms supported by the Content Addressable Storage. Valid values include:
>
> - ZSTD: Standard compression
>
> Support for IDENTITY (i.e., no compression) is implied.

That also allows us to get rid of `global.GetSupportedCompressors()` as well.